code/unclassified/src/TowerOfHanoi/Tower_of_Hanoi.c

```diff
@@ -0,0 +1,31 @@
+#include<stdio.h>
+
+void TowerOfHanoi(int n, char from_rod, char to_rod, char aux_rod);
+int
+main()

```

Comment:
Follow uniform Indentation across the entire commit. Otherwise, good effort!

Reply:
Thanks,Please have a look now.i have fixed it.